packages/docs/docs/tutorials/api-basics/create-fhir-data.md

```diff
@@ -2,6 +2,9 @@
 sidebar_position: 1
 ---
 
+import ExampleCode from '!!raw-loader!@site/../examples/src/tutorials/api-basics/create-fhir-data.ts';
```

Comment:
My assumption is that this file should be renamed to `.mdx` if it uses MDX syntax?

Reply:
Functionally, it works, but yeah maybe a good idea for consistency